weather.ozweather/resources/lib/weatherzone.py

```diff
@@ -340,9 +341,140 @@ def getWeatherData(urlPath, extendedFeatures = True, XBMC_VERSION=17.0):
         p = soup.find_all("p", class_="district-forecast")  
         weatherData["Current.ConditionLong"] = cleanLongDescription(p[0].text)
     except Exception as inst:
+        print(str(inst))
```

Comment:
`xbmc.log(...)` here and at all other places instead of `print`

Reply:
Those are for unit testing - previously fine with Ronie.  I can't use xmbc.log outside of xbmc for that...

Reply:
Also, been this way for literally years - like, in the Kodi repo since eden or so...

Reply:
Just thought it would be a good idea to adapt it while touching those lines, but if it okay for the others you can leave it as it is.

Reply:
Thanks - if there's a good solution to unit testing scrapers outside of Kodi (which makes my life vastly easier!) - whilst still having appropriate Kodi logging, I'm happy to do it...but I don't know of one.